src/harness/testcases/testdata/esc_sensor_record_0.json

```diff
@@ -1,8 +1,8 @@
 {
   "id": "esc_sensor/admin_0/esc_sensor_0",
   "installationParam": {
-    "latitude": 39.0119,
-    "longitude": -98.4842,
+    "latitude": 38.8,
```

Comment:
Any reason to make this change?

Reply:
this esc snesor is located exactly at the position of device_a, do you think it is realistic?
thank you

Reply:
Ahh I see now. In that case I think your change makes sense. Thanks for the explanation!